src/main/java/Duke.java

```diff
@@ -1,10 +1,357 @@
+        list.get(taskNum-1).markAsDone();
+        System.out.println("Nice! I've marked this task as done:");
+        System.out.println(DIVIDER + System.lineSeparator() + list.get(taskNum-1).toString()
+                + System.lineSeparator() + DIVIDER);
+    }
+
+    private static void unmarkTask(int taskNum) {
+        list.get(taskNum-1).markAsNotDone();
+        System.out.println("OK, I've marked this task as not done yet:");
+        System.out.println(DIVIDER + System.lineSeparator() + list.get(taskNum-1).toString()
+                + System.lineSeparator() + DIVIDER);
+    }
+
+    private static void acknowledgementMessage() {
+        String acknowledgement = DIVIDER + System.lineSeparator()
+                + "Got it. I've added this task: " + System.lineSeparator()
+                + list.get(list.size()-1).toString();
+        System.out.println(acknowledgement);
+        System.out.println("Now you have " + (list.size()) + " task(s) in the list."
+                + System.lineSeparator() + DIVIDER);
+    }
+
+    private static void deleteTask(int taskNum) {
+        String acknowledge = DIVIDER + System.lineSeparator() + "Noted. I've removed this task: "
+                + System.lineSeparator() + list.get(taskNum-1).toString() + System.lineSeparator()
+                + "Now you have " + (list.size()-1) + " tasks in the list." + System.lineSeparator() + DIVIDER;
+        System.out.println(acknowledge);
+        list.remove(taskNum-1);
+    }
+
+    //Update the file
+    private static void updateFile() {
+        for (Task currentTask : list) {
+            String statusNum = currentTask.getStatusNum();
+            String taskDesc = currentTask.getTask();
+            String taskType = currentTask.getTaskIcon();
+
+            switch (taskType) {
+            case TODO_ICON:
+                try {
+                    writeToFile(taskType + ":" + statusNum + ":"
+                            + taskDesc);
+                } catch (IOException e) {
+                    System.out.println(e.getMessage());
+                }
+                break;
+            case DEADLINE_ICON:
+                String deadlineBy = currentTask.getDeadlineBy();
+                try {
+                    writeToFile(taskType + ":" + statusNum + ":"
+                            + taskDesc + COMMAND_BY + deadlineBy);
+                } catch (IOException e) {
+                    System.out.println(e.getMessage());
+                }
+                break;
+            case EVENT_ICON:
+                String eventStart = currentTask.getEventStart();
+                String eventEnd = currentTask.getEventEnd();
+                try {
+                    writeToFile(taskType + ":" + statusNum + ":"
+                            + taskDesc + COMMAND_FROM + eventStart
+                            + COMMAND_TO + eventEnd);
+                } catch (IOException e) {
+                    System.out.println(e.getMessage());
+                }
```

Comment:
May want to consider abstracting these lines into submethods. Method might be lengthy in the future if you try to scale up your project to include more cases.